src/BrowserFetcher.ts

```diff
@@ -220,14 +220,29 @@ export class BrowserFetcher {
     if (!(await existsAsync(this._downloadsFolder)))
       await mkdirAsync(this._downloadsFolder);
     try {
-      await downloadFile(url, archivePath, progressCallback);
-      await install(archivePath, outputPath);
+      if (os.arch() === 'arm64') {
+        fs.stat('/usr/bin/chromium-browser', function (err, stats) {
+          if (stats === undefined) {
+            console.error(` The chromium binary is not available for arm64: `);
+            console.error(` If you are on Ubuntu,You can install with: `);
+            console.error(` \n apt-get install chromium-browser\n`);
```

Comment:
please could we tidy these up a bit? They all have a space at the beginning that I think can go.